docs/docs/usage.md

```diff
@@ -181,3 +179,20 @@ Triggers the build process of the specified module according to the implicit or
 | `-g (--mtad-gen)`   | Optional  | If the parameter is provided, the deployment descriptor `mtad.yaml` is generated by default in the current folder or in the folder configured by the `--target` parameter. <br> A module's `path` property in the generated `mtad.yaml` file points to the module's build results if this module was selected using the `--modules` option. <br><br> <b>Notes</b>:<ul><li>The selected module list specified using the `--module` option, does not affect the list of modules in the resulting `mtad.yaml` file. The `mtad.yaml` file is always generated according to the default Cloud MTA Builder settings, the `build-parameters` configurations in the `mta.yaml` file (e.g. `supported-platforms`), and the selected target platform.<li>By default, the `mtad.yaml` is generated for the `cf` target platform. You can configure a different target plaform using the `--platform` option.  | `mbt module-build -m=my_module1,my_module2 -g`
 | `-p (--platform)`   | Optional  |  The name of the target deployment platform. Used only with the `-g (--mtad-gen)` parameter. <br>The supported deployment platforms are: <ul><li>`cf` for SAP Cloud Platform, Cloud Foundry environment  <li>`neo` for the SAP Cloud Platform, Neo environment <li>`xsa` for the SAP HANA XS advanced model</ul> If this parameter is not provided, the `mtad.yaml` file is generated for the SAP Cloud Platform, Cloud Foundry environment.                             | `mbt module-build -m=my_module1,my_module2 -g -p=neo`
 
+
+&nbsp;
+
+### How to generate SBOM file from the project source (BETA)
+
+<b>`mbt sbom-gen`</b>
+
```

Comment:
Is this correct? 
This command generates SBOMs for project modules and merges into one, according to configurations in the MTA development descriptor (mta.yaml).
Merges what into one? Merges the SBOMs into one?

Reply:
the mbt sbom-gen command is correct.

all parameters of sbom-gen command have default values